teamcity: failed test: TestPlannerLogic

The following tests appear to have failed on master (test): TestPlannerLogic/local, TestPlannerLogic/local: TestPlannerLogic/local/show_trace, TestPlannerLogic

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestPlannerLogic).

[#1138761](https://teamcity.cockroachdb.com/viewLog.html?buildId=1138761):


```
TestPlannerLogic/local
--- FAIL: test/TestPlannerLogic: TestPlannerLogic/local (0.000s)




TestPlannerLogic/local: TestPlannerLogic/local/show_trace
...<semantic_type:INT width:64 precision:0 visible_type:BIGINT > nullable:false hidden:false > columns:<name:"v" id:2 type:<semantic_type:INT width:64 precision:0 visible_type:BIGINT > nullable:true hidden:false > next_column_id:3 families:<name:"primary" id:0 column_names:"k" column_names:"v" column_ids:1 column_ids:2 default_column_id:2 > next_family_id:1 primary_index:<name:"primary" id:1 unique:true column_names:"k" column_directions:ASC column_ids:1 foreign_key:<table:0 index:0 name:"" validity:Validated shared_prefix_len:0 on_delete:NO_ACTION on_update:NO_ACTION match:SIMPLE > interleave:<> partitioning:<num_columns:0 > type:FORWARD > next_index_id:3 privileges:<users:<user:"admin" privileges:2 > users:<user:"root" privileges:2 > > next_mutation_id:3 format_version:3 state:DROP draining_names:<parent_id:53 name:"kv" > view_query:"" drop_time:... replacement_of:<id:0 time:<> > audit_mode:DISABLED drop_job_id:... gc_mutations:<index_id:2 drop_time:... job_id:... > >
                    dist sender send  r6: sending batch 1 Put to (n1,s1):1
                    sql txn           rows affected: 0
                    dist sender send  r11: sending batch 8 QueryIntent to (n1,s1):1
                    dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    flow              Put /Table/3/1/54/2/1 -> table:<name:"kv" id:54 parent_id:53 version:8 modification_time:<wall_time:... > columns:<name:"k" id:1 type:<semantic_type:INT width:64 precision:0 visible_type:BIGINT > nullable:false hidden:false > columns:<name:"v" id:2 type:<semantic_type:INT width:64 precision:0 visible_type:BIGINT > nullable:true hidden:false > next_column_id:3 families:<name:"primary" id:0 column_names:"k" column_names:"v" column_ids:1 column_ids:2 default_column_id:2 > next_family_id:1 primary_index:<name:"primary" id:1 unique:true column_names:"k" column_directions:ASC column_ids:1 foreign_key:<table:0 index:0 name:"" validity:Validated shared_prefix_len:0 on_delete:NO_ACTION on_update:NO_ACTION match:SIMPLE > interleave:<> partitioning:<num_columns:0 > type:FORWARD > next_index_id:3 privileges:<users:<user:"admin" privileges:2 > users:<user:"root" privileges:2 > > next_mutation_id:3 format_version:3 state:DROP draining_names:<parent_id:53 name:"kv" > view_query:"" drop_time:... replacement_of:<id:0 time:<> > audit_mode:DISABLED drop_job_id:... gc_mutations:<index_id:2 drop_time:... job_id:... > >
                    dist sender send  r6: sending batch 1 Put to (n1,s1):1
                    sql txn           rows affected: 0
                    dist sender send  r11: sending batch 8 QueryIntent to (n1,s1):1
                    dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
                    dist sender send  r6: sending batch 1 PushTxn to (n1,s1):1
                    dist sender send  r11: sending batch 2 ResolveIntent to (n1,s1):1
                    
                
            logic.go:2345: 
                testdata/planner_test/show_trace:348: error while processing
            logic.go:2346: testdata/planner_test/show_trace:348: too many errors encountered, skipping the rest of the input



TestPlannerLogic
--- FAIL: test/TestPlannerLogic (10.880s)

------- Stdout: -------
    test_log_scope.go:81: test logs captured to: /tmp/logTestPlannerLogic088024876
    test_log_scope.go:62: use -show-logs to present logs inline




```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/planner_test/show_trace
+++ b/pkg/sql/logictest/testdata/planner_test/show_trace
@@ -15,6 +15,7 @@ query TT
 SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -28,10 +29,8 @@ flow              CPut /Table/3/1/53/2/1 -> database:<name:"t" id:53 privileges:
 dist sender send  querying next range at /Table/2/1/0/"t"/3/1
 dist sender send  r6: sending batch 2 CPut to (n1,s1):1
 sql txn           rows affected: 0
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/2/1/0/"t"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 
 
@@ -44,6 +43,7 @@ SELECT operation, regexp_replace(message, 'wall_time:\d+', 'wall_time:...') as m
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -67,10 +67,8 @@ dist sender send  r6: sending batch 2 CPut to (n1,s1):1
 dist sender send  querying next range at /Table/3/1/53/2/1
 dist sender send  r6: sending batch 1 Get to (n1,s1):1
 sql txn           rows affected: 0
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/2/1/53/"kv"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 
 # We avoid using the full trace output, because that would make the
@@ -90,7 +88,7 @@ WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
   /* since the transactions involved are multi-range, intent resolving is async and
   sometimes there's still intents around */
-  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent)%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -158,6 +156,7 @@ SELECT operation, regexp_replace(regexp_replace(message, 'wall_time:\d+', 'wall_
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -185,12 +184,10 @@ dist sender send  querying next range at /Table/3/1/53/2/1
 dist sender send  r6: sending batch 1 Get to (n1,s1):1
 table reader      fetched: /kv/primary/1/v -> /2
 flow              CPut /Table/55/1/...PK.../0 -> /TUPLE/1:1:Int/1/1:2:Int/2
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/55/1/...PK.../0
 dist sender send  r20: sending batch 1 CPut to (n1,s1):1
 dist sender send  querying next range at /Table/2/1/53/"kv2"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 flow              fast path completed
 sql txn           rows affected: 1
@@ -202,6 +199,7 @@ query TT
 SELECT operation, regexp_replace(message, '(\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.)?\d\d\d\d\d+', '...PK...') as message
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -242,6 +240,7 @@ SELECT operation,
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -290,7 +289,7 @@ WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
   /* since the transactions involved are multi-range, intent resolving is async and
   sometimes there's still intents around */
-  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent)%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -325,6 +324,7 @@ SELECT operation, regexp_replace(regexp_replace(regexp_replace(message, 'job_id:
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'

--- a/pkg/sql/opt/exec/execbuilder/testdata/show_trace
+++ b/pkg/sql/opt/exec/execbuilder/testdata/show_trace
@@ -11,6 +11,7 @@ query TT
 SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -24,10 +25,8 @@ flow              CPut /Table/3/1/53/2/1 -> database:<name:"t" id:53 privileges:
 dist sender send  querying next range at /Table/2/1/0/"t"/3/1
 dist sender send  r6: sending batch 2 CPut to (n1,s1):1
 sql txn           rows affected: 0
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/2/1/0/"t"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 
 
@@ -40,6 +39,7 @@ SELECT operation, regexp_replace(message, 'wall_time:\d+', 'wall_time:...') as m
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -63,10 +63,8 @@ dist sender send  r6: sending batch 2 CPut to (n1,s1):1
 dist sender send  querying next range at /Table/3/1/53/2/1
 dist sender send  r6: sending batch 1 Get to (n1,s1):1
 sql txn           rows affected: 0
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/2/1/53/"kv"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 
 # We avoid using the full trace output, because that would make the
@@ -86,7 +84,7 @@ WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
   /* since the transactions involved are multi-range, intent resolving is async and
   sometimes there's still intents around */
-  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent)%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -154,6 +152,7 @@ SELECT operation, regexp_replace(regexp_replace(message, 'wall_time:\d+', 'wall_
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -181,12 +180,10 @@ dist sender send  querying next range at /Table/3/1/53/2/1
 dist sender send  r6: sending batch 1 Get to (n1,s1):1
 table reader      fetched: /kv/primary/1/v -> /2
 flow              CPut /Table/55/1/...PK.../0 -> /TUPLE/1:1:Int/1/1:2:Int/2
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  querying next range at /Table/55/1/...PK.../0
 dist sender send  r20: sending batch 1 CPut to (n1,s1):1
 dist sender send  querying next range at /Table/2/1/53/"kv2"/3/1
 dist sender send  r8: sending batch 5 QueryIntent to (n1,s1):1
-dist sender send  querying next range at /Table/SystemConfigSpan/Start
 dist sender send  r6: sending batch 1 EndTxn to (n1,s1):1
 flow              fast path completed
 sql txn           rows affected: 1
@@ -198,6 +195,7 @@ query TT
 SELECT operation, regexp_replace(message, '(\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.)?\d\d\d\d\d+', '...PK...') as message
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -238,6 +236,7 @@ SELECT operation,
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -286,7 +285,7 @@ WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
   /* since the transactions involved are multi-range, intent resolving is async and
   sometimes there's still intents around */
-  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent)%'
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'
@@ -321,6 +320,7 @@ SELECT operation, regexp_replace(regexp_replace(regexp_replace(message, 'job_id:
   FROM [SHOW KV TRACE FOR SESSION]
 WHERE message NOT LIKE '%Z/%' AND message NOT LIKE 'querying next range at%'
   AND operation NOT LIKE 'kv.DistSender: sending partial batch%'  -- order of partial batches is not deterministic
+  AND message NOT SIMILAR TO '%(PushTxn|ResolveIntent|SystemConfigSpan)%'
   AND tag NOT LIKE '%intExec=%'
   AND tag NOT LIKE '%scExec%'
   AND tag NOT LIKE '%IndexBackfiller%'

